content/shared/influxdb3-admin/query-system-data/_index.md

```diff
@@ -0,0 +1,141 @@
+<!--Shortcode-->
+{{% product-name %}} stores data related to the database server, queries, and tables in _system tables_ within your cluster.
+You can query the system tables for information about your running server, databases, and and table schemas.
+
+## Query system tables
+
+- [Use the HTTP query API](#use-the-http-query-api)
+  - [Examples](#examples)
+    - [Show tables](#show-tables)
+    - [View column information for a table](#view-column-information-for-a-table)
+
+### Use the HTTP query API 
+
+Use the HTTP API `/api/v3/query_sql` endpoint to retrieve system information about your database server and table schemas in {{% product-name %}}.
+
+> [!Note]
+> {{% product-name %}} uses separate API endpoints for SQL and InfluxQL queries.
+> Both endpoints support the same parameters.
+> 
+> For more information about using InfluxQL, see [Query data with InfluxQL](/influxdb3/version/query-data/influxql/).
+
+To execute a query, send a `GET` or `POST` request to the endpoint:
+
+- `GET`: Pass parameters in the URL query string (for simple queries)
+- `POST`: Pass parameters in a JSON object (for complex queries and readability in your code)
+
+Include the following parameters:
+
+- `q`: _({{< req >}})_ The SQL query to execute.
+- `db`: _({{< req >}})_ The database to execute the query against.
+- `params`: A JSON object containing parameters to be used in a _parameterized query_.
+- `format`: The format of the response (`json`, `jsonl`, `csv`, `pretty`, or `parquet`).
+  JSONL (`jsonl`) is preferred because it streams results back to the client.
+  `pretty` is for human-readable output. Default is `json`.
+
+#### Examples
+
+> [!Note]
+> #### system_ sample data
```

Comment:
Does this underscore parse correctly without being escaped? Does it italicize the text?

```suggestion
> #### system\_ sample data
```